Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs.NetCore/GenerateSecurityGraphs.NetCore.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
+    <PackageReference Include="QuikGraph" Version="2.2.0" />
+    <PackageReference Include="QuikGraph.Graphviz" Version="2.2.0" />
+    <PackageReference Include="QuikGraph.Serialization" Version="2.2.2" />
```

Comment:
Why are you not using the Sandwych.QuickGraph.Core package?

Reply:
The package doesn't support DotEngine, so I found this package which works on minimal code changes.